### PR TITLE
"Visible label is part of accessible name" [2ee8b8]: add assumption on (no) missing resources

### LINKED ISSUE #1618: Visible label (2ee8b8) inconsistency between pass and inapplicable
Rule: https://act-rules.github.io/rules/2ee8b8
This is regarding "Passed example 6" (Pass6) and "Inapplicable example 4" (NA4). NA4 notes that it doesn't apply because the content is an image and not visible text. However, the argument that the rule passes for Pass6 is basically the same - it's not visible text. This seems inconsistent. If the content renders as an image and not text, is it a pass, or is it inapplicable?

Note that both situations /can/ result in rendered text, which makes the situation more difficult. If the font fails to load or isn't supported for Pass6, or if the image fails to load for NA4, the text will display in its place. In this scenario, both of these would be a failure.

### PR DESCRIPTION
**Pull request by Jym77**
Add an assumption that no resources are missing, to avoid corner case of images being rendered as their alt text.

Closes issue(s):

- closes #1618 

Need for Final Call:
This will require a 1 week Final Call (adding an assumption)

---

### **When creating PR:**

- [X] Make sure you're requesting to **pull a branch** (right side) to the `develop` branch (left side).
- [X] Make sure you do not remove the "How to Review and Approve" section in your pull request description

### **After creating PR:**

- [X] Add yourself (and co-authors) as "Assignees" for PR.
- [X] Add label to indicate if it's a `Rule`, `Definition` or `Chore`.
- [X] [Link the PR to any issue it solves](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue). This will be done automatically by [referencing the issue](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) at the top of this comment in the indicated place.
- [X] Optionally request feedback from anyone in particular by assigning them as "Reviewers".

### **When merging a PR:**

- [ ] Close any issue that the PR resolves. This will happen automatically upon merging if the PR was correctly linked to the issue, e.g. by referencing the issue at the top of this comment.

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Final Call period. In case of disagreement, the longer period wins.

### CODE REVIEW

File: _rules/visible-label-in-accessible-name-2ee8b8.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ For each target element, all [text nodes][] in the [visible text content][] eith
 
 ## Assumptions
 
-_There are currently no assumptions_
+This rule assumes that all resources needing for rendering the page are properly loaded. Checking if resources are missing is out of the scope of rules. Missing resources may be rendered as text (for example, missing `img` are rendered as their `alt` attribute), but this is not tested by this rule.
```
Comment (carlosapaduarte) [inline _rules/visible-label-in-accessible-name-2ee8b8.md:43]:
```suggestion
This rule assumes that all resources needed for rendering the page are properly loaded. Checking if resources are missing is out of the scope of rules. Missing resources may be rendered as text (for example, missing `img` are rendered as their `alt` attribute), but this is not tested by this rule.
```

**Review by carlosapaduarte — DISMISSED** [review]
Just an editorial fix

File: _rules/visible-label-in-accessible-name-2ee8b8.md [hunk 2]
```diff
@@ -42,7 +42,7 @@ For each target element, all [text nodes][] in the [visible text content][] eith
 
 ## Assumptions
 
-_There are currently no assumptions_
+This rule assumes that all resources needed for rendering the page are properly loaded. Checking if resources are missing is out of the scope of rules. Missing resources may be rendered as text (for example, missing `img` are rendered as their `alt` attribute), but this is not tested by this rule.
```
Comment (daniel-montalvo) [inline _rules/visible-label-in-accessible-name-2ee8b8.md:45]:
```suggestion
This rule assumes that all resources needed for rendering the page are properly loaded. Checking if resources are missing is out of the scope of rules. Missing resources may be rendered as text (for example, missing `img` are rendered as their `alt` attribute).
```

**Review by daniel-montalvo — COMMENTED** [review]
I just don't think we need to say this twice. Otherwise approving

**Comment by Jym77** [comment]
Call for Review ends on November 11th.

**Review by colabottles — APPROVED** [review]

**Review by JenniferChadwick — APPROVED** [review]

**Comment by Jym77** [comment]
Final Call has ended. Merging.

**Pull request merged**